app/templates/entry/client-entry.js

```diff
@@ -66,7 +66,7 @@ import electron from 'electron'
 Vue.prototype.$q.electron = electron
 <% } %>
 
-<% if (ctx.debug) { %>
+<% if (ctx.dev || ctx.debug) { %>
```

Comment:
This is not needed, since `ctx.debug` implies dev build or production build with --debug parameter

Reply:
There productionTip is now showing up in the console when you run quasar dev

Reply:
yes, it should be `false`, the way it was when it was just `if (ctx.dev)`

Reply:
What do you mean? 

I tested two scenarios:
if (ctx.debug) --> productiontip is shown
if (ctx.debug || cyx.dev) --> productiontip is hidden


process.env.DEBUGGING is set with the same setter. (ctx.debug || ctx.dev)
That's why i thought that it should be the same here. 

Do you know where the issue from the productiontip is coming from?

Reply:
You are right. Sorry about my own confusion. :)
Reopening...

Reply:
No problem!
I was thinking the same before I tested it:)

Do I also need to make an issue for this?

Reply:
no
https://discord.com/channels/415874313728688138/599146158828421140/768841045420408853